config/zones.json

```diff
@@ -3156,6 +3156,34 @@
     "flag_file_name": "pt.png",
     "timezone": null
   },
+  "RE": {
+    "bounding_box": [
+      [
+        54.374008,
+        -22.069087
+      ],
+      [
+        56.334630,
+        -20.421464
+      ]
+    ],
+    "_comment": "2 power plants are using bagasse (biomass) or coal -> 211MW, used at 15-18% with bagasse only in the year from July to November",
```

Comment:
This seems to be a comment about the electricity data (not the geometry). Can you move it to the parser?

Reply:
Fine I'll clean up that and the commented code when I have access to a PC, I forgot it.

Reply:
I can't give you an output for now. This is not my PC, I can't install python for now :/
Maybe in a week ;)

Reply:
Finally, I had the authorisation to install python on this PC.
So I made the correction, and I have an output.
The upload on github doesn't seems to work, so here's a framadrop link :
[test-parser-RE.txt](https://framadrop.org/r/QrzBrK1C0L#bBdX7W4nyjHSRcubnF+abMS2engOHZR2T3AWQF9a3XI=)